libs/ng-strapi-auth/src/lib/core/services/auth/auth.service.ts

```diff
@@ -1,9 +1,59 @@
 import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { BehaviorSubject } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { LocalStorageKeyEnum } from '@ng-strapi-auth/ng-strapi-auth';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
 export class AuthService {
 
-  constructor() { }
+    authToken: string | null;
+    authApiBase: string = 'http://localhost:1337';
+    currentUser$: BehaviorSubject<any>;
```

Comment:
Also see how we extend the userModel with app-specific model and T from the xevlabs-auth-lib